client/my-sites/earn/ads/wrapper.jsx

```diff
@@ -284,101 +296,47 @@ class AdsWrapper extends Component {
 				{ isAllowedSection ? component : <FeatureExample>{ component }</FeatureExample> }
 			</>
 		);
-	}
-
-	render() {
-		const {
-			canAccessAds,
-			canActivateWordadsInstant,
-			canUpgradeToUseWordAds,
-			isEnrolledWithIneligiblePlan,
-			isWordadsInstantEligibleButNotOwner,
-			site,
-			siteId,
-			translate,
-		} = this.props;
+	};
 
-		let component = this.props.children;
+	const getComponentAndNotice = () => {
```

Comment:
I moved the logic for setting the component and notice, which had been inside the render() method for the class, to this method, which is then invoked to get the notice and component in the return call for the component on lines 336 and 337.